sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InheritedCollidingInterfaceMembers.AfterCSharp8.cs

```diff
@@ -0,0 +1,298 @@
+    {
+    }
+
+    public interface IAbstractMethodFirst
+    {
+        public abstract string AbstractMethod(string value);
+    }
+
+    public interface IAbstractMethodSecond
+    {
+        public abstract string AbstractMethod(string value);
+//                             ^^^^^^^^^^^^^^ Secondary
+    }
+
+    public interface IAbstractMethodCommon : IAbstractMethodFirst, IAbstractMethodSecond // Noncompliant {{Rename or add member 'AbstractMethod(string)' to this interface to resolve ambiguities.}}
+//                   ^^^^^^^^^^^^^^^^^^^^^
+    {
+    }
+
+    public interface IVirtualMethodFirst
+    {
+        public virtual string Virtual(string value)
+        {
+            return value;
+        }
+    }
+
+    public interface IVirtualMethodSecond
+    {
+        public virtual string Virtual(string value)
+//                            ^^^^^^^ Secondary
+        {
+            return value;
+        }
+    }
+
+    public interface IVirtualMethodCommon : IVirtualMethodFirst, IVirtualMethodSecond // Noncompliant {{Rename or add member 'Virtual(string)' to this interface to resolve ambiguities.}}
+//                   ^^^^^^^^^^^^^^^^^^^^
+    {
+    }
+
+    public interface IInternalMethodFirst
+    {
+        internal string Internal(string value)
+        {
+            return value;
+        }
+    }
+
+    public interface IInternalMethodSecond
+    {
+        internal string Internal(string value)
+//                      ^^^^^^^^ Secondary
+        {
+            return value;
+        }
+    }
+
+    public interface IInternalMethodCommon : IInternalMethodFirst, IInternalMethodSecond // Noncompliant {{Rename or add member 'Internal(string)' to this interface to resolve ambiguities.}}
+//                   ^^^^^^^^^^^^^^^^^^^^^
+    {
+    }
+
+    public interface IProtectedMethodFirst
+    {
+        protected string Protected(string value)
+        {
+            return value;
+        }
+    }
+
+    public interface IProtectedMethodSecond
+    {
+        protected string Protected(string value)
+//                       ^^^^^^^^^ Secondary
+        {
+            return value;
+        }
+    }
+
+    public interface IProtectedMethodCommon : IProtectedMethodFirst, IProtectedMethodSecond // Noncompliant {{Rename or add member 'Protected(string)' to this interface to resolve ambiguities.}}
+//                   ^^^^^^^^^^^^^^^^^^^^^^
+    {
+    }
+
+    public interface IPrivateMethodFirst
+    {
+        private string Private(string value)
+        {
+            return value;
+        }
+    }
+
+    public interface IPrivateMethodSecond
+    {
+        private string Private(string value)
+//                     ^^^^^^^ Secondary
+        {
+            return value;
+        }
+    }
+
+    public interface IPrivateMethodCommon : IPrivateMethodFirst, IPrivateMethodSecond // Noncompliant - FP: Issue is raised but since the methods are not accessible on the derived classes or interfaces it doesn't make sense.
```

Comment:
would it make sense to open an FP ticket for this specific issue (it would be easier to track in the MMF)

Reply:
I update the rules status here https://docs.google.com/spreadsheets/d/1RMc5s9rK3Q2CHKWp4MT-W7bM7zyTHQElAWa11zbUSkE and at the end of the sprint we will decide together how we will proceed further.